app/views/tags/index.html.erb

```diff
@@ -20,8 +20,9 @@
 <% end %>
 
 <main id="main-content" class="crayons-layout crayons-layout--1-col">
-  <header class="px-2 m:px-0">
+  <header class="px-2 m:px-0 flex items-center justify-between">
     <h1>Top tags</h1>
+    <a class="crayons-btn crayons-btn--ghost" href="<%= dashboard_following_tags_path %>">Following tags</a>
```

Comment:
We should probably use the rails link_to method, something like this: 

```
<%= link_to "Following tags", dashboard_following_tags_path, class: "crayons-btn crayons-btn--ghost" %>
```

Reply:
👍 i've pushed the change!